Vagrantfile

```diff
@@ -10,7 +10,7 @@ Vagrant.configure('2') do |config|
   config.vm.define 'ubuntu16.local' do |machine|
 
     machine.vm.box = "bento/ubuntu-16.04"
-    machine.vm.network :private_network, ip: '192.168.88.22'
+    machine.vm.network :private_network, ip: '192.168.88.10'
```

Comment:
Generally reorganised the Vagrant file, to group the same family of hosts together, and to give space in the IP address range for more.